remoulade/state/backends/redis.py

```diff
@@ -26,25 +26,42 @@ def __init__(self, *, namespace="remoulade-state", encoder=None, client=None, ur
         super().__init__(namespace=namespace, encoder=encoder)
         if url:
             parameters["connection_pool"] = redis.ConnectionPool.from_url(url)
+        parameters["decode_responses"] = True
```

Comment:
I think we should keep the encoder (and not use this). You should use the encoder to encode/decode each value of the state before the calls to hmset/after the call to hgetall.